crates/aggchain-proof-core/Cargo.toml

```diff
@@ -13,4 +13,4 @@ tiny-keccak = { git = "https://github.com/sp1-patches/tiny-keccak", tag = "patch
     "keccak",
 ] }
 
-sp1-zkvm = { version = "=4.0.0", features = ["verify"] }
+sp1-zkvm = { version = "=4.0.1", features = ["verify"] }
```

Comment:
We can not update `sp1-zkvm` version, even the patch fixes without fully understanding the changes and asserting the compatibility with the rest of the ecosystem.